cmd/version.go

```diff
@@ -0,0 +1,21 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// versionCmd represents the version command
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Displays the version of the currently installed Git Town executable",
+	Long:  `Displays the version of the currently installed Git Town executable`,
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("Git Town 4.0.0")
```

Comment:
This file defines the only command that the Go version supports right now, `version`. The logic for it is on this line, it prints this string here.